chat/documents.py

```diff
@@ -30,27 +27,23 @@ class Room(Document):
         regex=NAME_REGEX,
         unique=True
     )
-    participants = ListField(ReferenceField('UserDocument'))
+    is_group = BooleanField(default=False)
+    participants = ListField(StringField())
     created_at = DateTimeField(default=lambda: datetime.now(timezone.utc))
     updated_at = DateTimeField(default=lambda: datetime.now(timezone.utc))
 
+    meta = {"collection": "rooms"}
+
     def clean(self):
         """
         Validate room data before saving:
         - Remove duplicate participants.
-        - Limit participants to 50.
-        - Ensure room name matches allowed pattern.
+        - Limit participants to MAX_PARTICIPANTS.
         """
-        seen = set()
-        deduped_participants = []
-        for user in self.participants:
-            if user.id not in seen:
-                deduped_participants.append(user)
-                seen.add(user.id)
-        self.participants = deduped_participants
+        self.participants = list(set(self.participants))
```

Comment:
Deduplication uses set conversion, which changes participant order. Preserve order for predictable API results.
```
self.participants = list(dict.fromkeys(self.participants))
```